### DISCUSSION TITLE
github release and pypi releases

### DISCUSSION THREAD

**Discussion by derekschrock** [python-gitlab/python-gitlab#2708] [General]
The question here on Gitter https://app.gitter.im/#/room/#python-gitlab_Lobby:gitter.im/$ZiH8xxaBDBnoEegm7g_5Iu0uuAOr5ncundxoxdzvJZ4   Should 4.x+ tags be creating releases?  pypi is still 3.15.0 and github doesn't have the versioned release.

It seems the "Python Semantic Release" step need's an id such that the steps are not skipped?

**Comment by nejch** [comment]
@derekschrock yes, there was an issue with copy/paste there when I migrated to v8, sorry! but I also feel like we should first fix its changelog generation as it dumps all of the commit messages now and is a bit too noisy. Sorry for the delay!

**Reply by nejch** [reply]
@derekschrock pushed manually for now, let me know if there are any issues.
